$NetBSD: patch-simd_Makefile.am,v 1.1 2014/08/07 09:06:08 thomasklausner Exp $

Add libtool --tag so libtool knows what kind of stuff it compiles.

--- simd/Makefile.am.orig	2014-07-29 14:12:48.000000000 +0000
+++ simd/Makefile.am
@@ -67,7 +67,7 @@ endif
 AM_CPPFLAGS = -I$(top_srcdir) 
 
 .asm.lo:
-	$(AM_V_GEN) $(LIBTOOL) $(AM_V_lt) --mode=compile $(srcdir)/nasm_lt.sh $(AM_V_lt) $(NASM) $(NAFLAGS) -I$(srcdir) -I. $< -o $@
+	$(AM_V_GEN) $(LIBTOOL) $(AM_V_lt) --tag=CC --mode=compile $(srcdir)/nasm_lt.sh $(AM_V_lt) $(NASM) $(NAFLAGS) -I$(srcdir) -I. $< -o $@
 
 jsimdcfg.inc: $(srcdir)/jsimdcfg.inc.h ../jpeglib.h ../jconfig.h ../jmorecfg.h
 	$(AM_V_GEN) $(CPP) -I$(top_builddir) -I$(top_builddir)/simd $(srcdir)/jsimdcfg.inc.h | $(EGREP) "^[\;%]|^\ %" | sed 's%_cpp_protection_%%' | sed 's@% define@%define@g' > $@
